invenio_previewer/extensions/audio_videojs.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2023 Northwestern University.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Render audio player using videojs."""
+
+from flask import current_app, render_template
+
+from ..proxies import current_previewer
+from ..utils import dotted_exts
+
+# The audio file world is complex (but less so than video one), because support
+# depends on hardware, OS, software and browser. A reasonable choice of
+# supported extensions
+# (https://en.wikipedia.org/wiki/HTML5_audio#Supported_audio_coding_formats)
+# that are least likely lead to an unsupported warning for end-users are mp3,
+# wav, aac and flac, but this could be made changeable per instance.
+previewable_extensions = current_app.config.get(
+    "PREVIEWER_AUDIO_EXTS", ["mp3", "wav", "aac", "flac"]
```

Comment:
The hardcoded lists of supported file extensions might limit flexibility, how about considering making it more configurable so it can be overridable on the instance level, WDYT?

Reply:
I mean... it is explicitly made overridable by instance since we are using the `current_app.config.get` pattern here :smile: . The list here is just the default, if no additional configuration is set.

Reply:
Got it,  I appreciate the clarification on the configurability, maybe I didn't explain myself the right way🙂
However, my thought was about making these configurations more visible,  by placing them directly in the package's main [configuration ](https://github.com/inveniosoftware/invenio-previewer/blob/master/invenio_previewer/config.py#L35C25-L35C25) file instead of being buried within the function.

Reply:
I'd say because the previewers have previously been structured to be independent, that it makes more sense to keep the config here.